app/views/devise/shared/_flashes.html.erb

```diff
@@ -5,7 +5,10 @@
 <% end %>
 
 <% if alert %>
-  <div class="alert">
-    <%= alert %>
+  <div class="alert-container" data-controller="hide-devise-alert" data-hide-devise-alert-target="alert">
+    <div class="alert">
+      <p>Ops, something went wrong <i class="fa-regular fa-face-sad-tear"></i></p>
+      <%= alert %>
+    </div>
   </div>
 <% end %>
```

Comment:
What about the notices ?

Reply:
I will donit soon, i will send the screenshot in a bit